tests/testthat/test-connections.R

```diff
@@ -1,5 +1,27 @@
 context("catmaid login and get/post")
 
+test_that("set and get the environmental variables responsible for connections", {
+  #check the named object has all necessary fields..
+  expect_named(catmaid_connection_getenv(), c("server", "authname", "authpassword","token"))
```

Comment:
Will this work if no environment variables are set?

Reply:
In the latest commit, I have modified the test cases, such that the case will only run if all the env vars are set otherwise it skips the case with a message. I have also added a new object that is at the top where we can easily configure the public server url in the future.